recipes/flax/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "flax" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e995c4a40d4790751fbe08a44d7afec6fb22ac3e88a5f7912c40130cef81a3fa
+
+build:
+  skip: true  # [win or py<36]
```

Comment:
It looks like this package is pure python? If so, you can remove the skip, add `noarch: python` here and then change the dependencies on Python to `python >=3.6`.

What's the need to skip on Windows? If it's just a dependency is unavailable, we can just ignore that.

Reply:
it was an unavailable dependency. I addressed the comment by making the recipe noarch.

Reply:
Any idea why the windows build is failing with `conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64` ? 
Jaxlib is not available for windows so it should skip it.

Reply:
That's the error it spits out when a dependency is not available. It doesn't skip, it just fails CI--but we can ignore to merge.